electron/js/certutils.js

```diff
@@ -20,58 +20,89 @@
 'use strict';
 
 const config = require('./config');
-
-const asn1js = require('asn1js');
 const crypto = require('crypto');
-const pkijs = require('pkijs');
+const rs = require('jsrsasign');
 
-const MAIN_CERT = 'ff4880f09cc1e2d0edc0d07fdcdc987c2b5f4d9c';
+const MAIN_FP = '3pHQns2wdYtN4b2MWsMguGw70gISyhBZLZDpbj+EmdU=';
+const ALGORITHM_RSA = '2a864886f70d010101';
+const DIGICERT_EV_ROOT='-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAxszlc+b71LvlLS0ypt/l\ngT/JzSVJtnEqw9WUNGeiChywX2mmQLHEt7KP0JikqUFZOtPclNY823Q4pErMTSWC\n90qlUxI47vNJbXGRfmO2q6Zfw6SE+E9iUb74xezbOJLjBuUIkQzEKEFV+8taiRV+\nceg1v01yCT2+OjhQW3cxG42zxyRFmqesbQAUWgS3uhPrUQqYQUEiTmVhh4FBUKZ5\nXIneGUpX1S7mXRxTLH6YzRoGFqRoc9A0BBNcoXHTWnxV215k4TeHMFYE5RG0KYAS\n8Xk5iKICEXwnZreIt3jyygqoOKsKZMK/Zl2VhMGhJR6HXRpQCyASzEG7bgtROLhL\nywIDAQAB\n-----END PUBLIC KEY-----';
 const strip = (url) => url.replace(/https:|[\/]+/g, '');
 const pins = [
-  {url: strip(config.PROD_URL), keys: ['c85f71715e51593828e37cd8450501f3fa5fa4a2', MAIN_CERT]},
-  {url: 'wire.com', keys: [MAIN_CERT]},
-  {url: 'www.wire.com', keys: [MAIN_CERT]},
-  {url: 'prod-nginz-https.wire.com', keys: [MAIN_CERT]},
-  {url: 'prod-nginz-ssl.wire.com', keys: [MAIN_CERT]},
-  {url: 'prod-assets.wire.com', keys: [MAIN_CERT]},
+  {
+    url: strip(config.PROD_URL),
+    publicKeyInfo: {
+      algorithmID: ALGORITHM_RSA,
+      algorithmParam: null,
+      fingerprints: ['bORoZ2vRsPJ4WBsUdL1h3Q7C50ZaBqPwngDmDVw+wHA=', MAIN_FP],
+      issuerRootPubkeys: [DIGICERT_EV_ROOT],
+    },
+  },
+  {
+    url: 'wire.com',
+    publicKeyInfo: {
+      algorithmID: ALGORITHM_RSA,
+      algorithmParam: null,
+      fingerprints: [MAIN_FP],
+    },
+  },
+  {
+    url: 'www.wire.com',
+    publicKeyInfo: {
+      algorithmID: ALGORITHM_RSA,
+      algorithmParam: null,
+      fingerprints: [MAIN_FP],
+    },
+  },
+  {
+    url: 'prod-nginz-https.wire.com',
```

Comment:
The internal environments are not pinned. app.wire.com is defined in `config.PROD_URL`, but I will change this, too.